pkg/crawl/crawler.go

```diff
@@ -91,7 +91,7 @@ func (c Crawler) Crawl(ctx context.Context, root string, match MatchFunc, proces
 		if err != nil {
 			svc1log.FromContext(ctx).Warn("Error processing path",
 				svc1log.Stacktrace(err),
-				svc1log.UnsafeParam("path", path),
+				svc1log.UnsafeParam("pathOnDisk", path),
```

Comment:
This rename is to avoid `path` param clash against the relative path inside a zip.